### DISCUSSION TITLE
conditional relationship picker

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#8185] [Help] [answered]
Hi everyone.
I want to have a relationship picker, i mean that their values ​​depend on the value of another drop-down list. as the example of countries and cities

ps : I tried to add a data provider to the dropdown list and I want the filter to be based on a relational column, but I have no possibility to add this column

**Comment by melohagan** [comment, marked as answer]
Hey @rafikApp 

At the moment there is no way to filter the options presented by the relationship picker. There is an open issue here: https://github.com/Budibase/budibase/issues/7810

**Reply by gerickt** [reply]
any alternative?
